be/src/formats/parquet/group_reader.cpp

```diff
@@ -286,6 +279,51 @@ Status GroupReader::_read_range(const std::vector<int>& read_columns, const Rang
     return Status::OK();
 }
 
+StatusOr<size_t> GroupReader::_read_range_round_by_round(const Range<uint64_t>& range, Filter* filter,
+                                                         ChunkPtr* chunk) {
+    const std::vector<int>& round_order = _round_by_round_ctx->get_round_order();
+    size_t round_cost = 0;
+    size_t hit_count = 0;
+    for (int col_idx : round_order) {
+        auto& column = _param.read_cols[col_idx];
+        round_cost += _round_by_round_ctx->get_column_cost(col_idx);
```

Comment:
should we get cost from column reader which can record cpu cost

Reply:
We left comment to optimize